roofit/roofit/test/testRooExponential.cxx

```diff
@@ -87,6 +88,12 @@ TEST(RooExponential, Integral)
 
 
 TEST(RooExponential, IO) {
+  // Suppress all streamer info warnings. Revert if possible!
+  ROOTUnitTestSupport::CheckDiagsRAII diags;
+  diags.optionalDiag(kWarning, "TStreamerInfo::BuildCheck", "Do not try to write objects with the current class definition", false);
+  diags.optionalDiag(kWarning, "TStreamerInfo::CompareContent", "The following data member of", false);
+  diags.optionalDiag(kWarning, "TClass::Init", "no dictionary for class", false);
```

Comment:
Do we have a GitHub issue for those?

Reply:
Not yet. It's commit https://github.com/root-project/root/pull/5591/commits/0fb3d106728d50d38fdc9490dcc87db892a9bb43

but I wanted to open the GH issues once it has been rebased on master, so I can open one issue per commit.